src/main/PdfLittleSigner/PdfSigner.cs

```diff
@@ -22,9 +29,10 @@ public class PdfSigner : IPdfSigner
 
         private readonly string _outputPdfFileString = "";
         private Stream _outputPdfStream;
-        private readonly Size _imageSize = new(248, 99);
-        private readonly Size _imageLocation = new(60, 100);
+        private readonly Size _imageSize = new(150, 150);
         private readonly ILogger<PdfSigner> _logger;
+        private string[] _unsupportedImageFormats = { ".png" };
```

Comment:
dlaczego png jest nie supportowany??

Reply:
z tego co rozumiem to nie jest wspierany iText7 nie wiem czy to taki ich wymysł czy pdf nie wspiera png, teraz przy przeskalowaniu jest zapisywany jako jpg to powinno się dać go obsłużyć, przetestuję to i ewentualnie usunę jeśli nie będzie potrzebne

Reply:
teraz jesteśmy w stanie obsłużyć png